src/Microsoft.Data.SqlClient/tests/ManualTests/Microsoft.Data.SqlClient.ManualTesting.Tests.csproj

```diff
@@ -59,7 +59,9 @@
     <Compile Include="AlwaysEncrypted\TestTrustedMasterKeyPaths.cs" />
     <Compile Include="DataCommon\AADUtility.cs" />
     <Compile Include="DataCommon\CheckConnStrSetupFactAttribute.cs" />
-    <Compile Include="EventSource\EventSourceTest.cs" />
+    <Compile Condition="'$(TargetGroup)'=='netcoreapp'" Include="TracingTests\DiagnosticTest.cs" />
```

Comment:
Please add these 2 entries to a new `ItemGroup` after this current `ItemGroup` as under:
```xml
<ItemGroup Condition="'$(TargetGroup)'=='netcoreapp'">
  <Compile Include="TracingTests\DiagnosticTest.cs" />
  <Compile Include="TracingTests\FakeDiagnosticListenerObserver.cs" />
</ItemGroup
```

Reply:
done.